pkg/systemd/generate/containers.go

```diff
@@ -163,6 +183,21 @@ func generateContainerInfo(ctr *libpod.Container, options entities.GenerateSyste
 		startTimeout = *options.StartTimeout
 	}
 
+	var wants *[]string
+	if options.Wants != wants {
+		wants = options.Wants
+	}
+
+	var after *[]string
+	if options.After != after {
+		after = options.After
+	}
+
+	var requires *[]string
+	if options.Requires != requires {
+		requires = options.Requires
+	}
+
```

Comment:
Thanks for your comment. Unfortunately, I don't know what `NOP` means, and I can't remove the whole piece of code. But I tried to update this code according to the new definitions and now it looks so:
```go
	var wants []string
	if options.Wants != nil {
		wants = options.Wants
	}

	var after []string
	if options.After != nil {
		after = options.After
	}

	var requires []string
	if options.Requires != nil {
		requires = options.Requires
	}

````

Reply:
https://en.wikipedia.org/wiki/NOP_%28code%29

This code does nothing useful, you can just use `options.{Wants,After,Requires}` below on line 239-241 instead of defining another slice.

Reply:
@Luap99 Thanks a ton! Fixed